packages/ember-toucan-core/src/components/form/fields/input.hbs

```diff
@@ -35,21 +35,37 @@
         {{/if}}
       </field.Hint>
     {{/if}}
-
     <field.Control class="mt-1.5 flex">
       <Form::Controls::Input
         id={{field.id}}
         aria-describedby="{{if @hint field.hintId}} {{if @error field.errorId}}"
         aria-invalid={{if @error "true"}}
+        class="w-full"
```

Comment:
Do we need to add `w-full` here?  I'm just wondering - consumers may want their own widths and as you know, overriding CSS classes with Tailwind can be a pain.  Whereas if we let folks decide what they want to do instead by passing in `class` it gives them more control.


If we want this demo to be full width, we could add the following to the demo instead of here like:

```hbs
{{! demo above }}
<Form::Fields::Input class="w-full" @label='Label' />
```